app/split_integer.py

```diff
@@ -1,7 +1,11 @@
 def split_integer(value: int, number_of_parts: int) -> list:
     parts = []
+    number = value
```

Comment:
There is a missing divisor in the assignment of `number`. You need to divide `value` by `number_of_parts` to initialize `number` correctly.